content/3_Silver/Prefix_Sums.mdx

```diff
@@ -681,6 +692,305 @@ as expected.
 
 Since no matter the size of the submatrix we are summing, we only need to access four values of the 2D prefix sum array, this runs in $\mathcal{O}(1)$ per query after an $\mathcal{O}(NM)$ preprocessing.
 
+## Creation From Endpoints
+
+<FocusProblem problem={problems.sample3[0]} />
+
+### 1-Dimensional
+
+A common application of prefix sums is to generate a prefix sum array from given interval endpoints in the form $[a, b]$
+
+We are given the problem that for many such intervals, compute the point with the most overlapping intervals. 
+
+Naïvely, we can compute the interval by looping through each indice from $[a,b]$ individually, resulting in an $\mathcal{O}(NQ)$ time complexity (where N is the maximum size of our interval and Q is the number of queries), which easily times out (think what happens when the interval [0, N] is queried many times).
+
+We can do better. It's easy to see that an increment of $x$ (before computation) in $\texttt{prefix}[i]$ causes all subsequent $\texttt{prefix}[i...N]$ (after computation) to increase by $x$. This concept can be conceptualized through increment and decrement points. An increment point increases (and decrement decreases) all subsequent cells. (Note that our decrement point is located at $B+1$ because the interval is inclusive.)
```

Comment:
This is also pointed out below by you and Harry, but please distinguish arr and prefix/pfx. 

In this case: "An increment of $x$ in $\texttt{arr}[i]$ causes all of $\texttt{prefix}[i...N]$ ..."